apps/verticalhome/js/pin_app_navigation.js

```diff
@@ -0,0 +1,301 @@
+'use strict';
+/* global ApplicationSource */
+/* global BookmarkSource */
+/* global CollectionSource */
+/* global configurator */
+/* global dispatchEvent */
+/* global GaiaGrid */
+
+(function(exports) {
+
+
+	/**
+	 * This function as a constructor of PinAppNavigation. It gets dom element that contains elements
```

Comment:
It's really good practice to document main methods, but could you please rephrase this comment to make it less confusing